dietpi/patch_file

```diff
@@ -426,7 +426,7 @@ _EOF_
 			# + Force auto swapfile size https://github.com/Fourdee/DietPi/issues/1593#issuecomment-371516418
 			/DietPi/dietpi/func/dietpi-set_dphys-swapfile 1
 
-			G_AGP dphys-swapfile
+			apt-mark auto dphys-swapfile
```

Comment:
I believe this should be a forced removal, to prevent it from remaining (in rare case?) installed and conflicting with our swapfile system?

Reply:
@Fourdee 
I checked the priority of all packages and only use auto mark for optional, extra etc., which includes dphys-swapfile. But yeah, not sure if this can vary on systems, properly better G_AGP to be on secure side.

Reply:
Done: https://github.com/Fourdee/DietPi/pull/1994/commits/b60e8fbd45b770edfa9eacd8a11f9e93235ee436